specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -1097,17 +1097,1048 @@
+        ],
+        "x-ms-examples": {
+          "Get large person group's training status example": {
+            "$ref": "./examples/GetLargePersonGroupTrainingStatus.json"
+          }
+        }
+      }
+    },
+    "/largepersongroups": {
+      "get": {
+        "description": "List large person groups and their information.",
+        "operationId": "LargePersonGroup_List",
+        "parameters": [
+          {
+            "name": "start",
+            "in": "query",
+            "required": false,
+            "description": "List large person groups from the least largePersonGroupId greater than the \"start\".",
+            "type": "string",
+            "maxLength": 64
+          },
+          {
+            "name": "top",
+            "in": "query",
+            "required": false,
+            "description": "The number of large person groups to list.",
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 1000,
+            "default": 1000
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call returns an array of large person groups and their information.",
+            "schema": {
+              "$ref": "#/definitions/LargePersonGroups"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List large person groups example": {
+            "$ref": "./examples/ListLargePersonGroups.json"
+          }
+        }
+      }
+    },
+    "/largepersongroups/{largePersonGroupId}/train": {
+      "post": {
+        "description": "Queue a large person group training task, the training task may not be started immediately.",
+        "operationId": "LargePersonGroup_Train",
+        "parameters": [
+          {
+            "$ref": "#/parameters/largePersonGroupId"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "202": {
+            "description": "The training task was queued successfully."
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Queue large person group training": {
+            "$ref": "./examples/QueueLargePersonGroupTraining.json"
+          }
+        }
+      }
+    },
+    "/largepersongroups/{largePersonGroupId}/persons/{personId}/persistedfaces": {
+      "post": {
+        "description": "Add a representative face to a person for identification. The input face is specified as an image with a targetFace rectangle.",
+        "operationId": "LargePersonGroupPerson_AddPersonFaceFromUrl",
```

Comment:
Seems this is not consistent with `LargePersonGroupPerson_Delete/Get/UpdateFace`.

Reply:
Fixed it in this PR since it is tiny and new PR does take longer.